relayer/pkg/starknet/starknet.go

```diff
@@ -119,10 +122,10 @@ func (t *Test) DeployEnv(nodes int) {
 // Sets up the starknet client
 func (t *Test) SetupClients() {
 	t.sc, err = t.NewStarkNetDevnetClient(&blockchain.EVMNetwork{
-		Name: "starknet-dev",
+		Name: serviceKeyL2,
 		URL:  t.Env.URLs[serviceKeyL2][1],
 		PrivateKeys: []string{
-			"c4da537c1651ddae44867db30d67b366",
+			defaultWalletPrivKey,
 		},
 	})
```

Comment:
How did this code end up in the `relayer` module? :)

We should move it to the `integration-tests` module.

Reply:
Adjusted and moved